files/source/galaxy_workflows.yml

```diff
@@ -0,0 +1,15 @@
+workflows:
+  - trs_id: "https://dockstore.org/api/ga4gh/trs/v2/tools/#workflow/github.com/iwc-workflows/rnaseq-pe/main/versions/v0.9"
```

Comment:
I think the actual trsID here is just `#workflow/github.com/iwc-workflows/rnaseq-pe/main`, though it does bring up how we want to pin specific versions (or not).  Do we want an explicit extra step to update this annotation as new versions are released, or do we want 'latest' @mvdbeek

Reply:
the word 'latest' scared me :fearful:

Reply:
> I think the actual trsID here is just `#workflow/github.com/iwc-workflows/rnaseq-pe/main`, though it does bring up how we want to pin specific versions (or not)

No, this is the full TRS id as we need it. No, no latest, we should always use pinned versions.